src/applications/financial-status-report/components/shared/NeedsToVerifyAlert.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useEffect } from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+import VerifyAlert from 'platform/user/authorization/components/VerifyAlert';
+import recordEvent from 'platform/monitoring/record-event';
+
+export const heading =
+  'You’ll need to verify your identity to access more VA.gov tools and features';
+
+const NeedsToVerify = () => {
+  const alertRef = useRef(null);
+
+  useEffect(
+    () => {
+      if (alertRef?.current) {
+        focusElement(alertRef.current);
+      }
+    },
+    [alertRef],
+  );
+
+  recordEvent({
+    event: 'visible-alert-box',
+    'alert-box-type': 'continue',
+    'alert-box-heading': heading,
+    'error-key': 'not_verified',
+    'alert-box-full-width': false,
+    'alert-box-background-only': false,
+    'alert-box-closeable': false,
+    'reason-for-alert': `Not verified`,
+  });
```

Comment:
Do we have the tracking set up on the GA side for this? (might be chill to have a separate ticket).

Reply:
Yeah, I don’t think we do, but we probably should spin one up for analytics. Do you see any harm in rolling it out and talking to the GA team as part of a secondary ticket? 

I don't think it will though anything off as far as I know